python/paddle/v2/__init__.py

```diff
@@ -37,9 +38,26 @@
 import image
 
 __all__ = [
-    'optimizer', 'layer', 'activation', 'parameters', 'init', 'trainer',
-    'event', 'data_type', 'attr', 'pooling', 'data_feeder', 'dataset', 'reader',
-    'topology', 'networks', 'infer', 'plot', 'evaluator', 'image'
+    'optimizer',
+    'layer',
+    'activation',
+    'parameters',
+    'init',
+    'trainer',
+    'event',
+    'data_type',
+    'attr',
+    'pooling',
+    'data_feeder',
+    'dataset',
+    'reader',
+    'topology',
+    'networks',
+    'infer',
+    'plot',
+    'evaluator',
+    'image',
+    'master',
```

Comment:
Because when user use the package, it will be
```python
import paddle.v2 as paddle
client = paddle.master.client("localhost:3000", 30)
record = client.NextRecord()
```
I was thinking since there is a `client` after package name (`paddle.master.client`), maybe it's clear that it means master's client?

I am actually not very familiar with Python's naming convention, do you have any suggestion?